core/components/molecules/stepper/Stepper.tsx

```diff
@@ -80,9 +83,10 @@ export const Stepper = (props: StepperProps) => {
 
         const { label, value } = step;
 
+        const skippedIndex = skipIndexes.includes(index);
         const activeStep = active === index;
-        const completedStep = completed >= index;
-        const disabled = completed + 1 < index;
+        const completedStep = completed >= index && !(skippedIndex);
+        const disabled = completed + 1 < index && !(skippedIndex);
```

Comment:
Condition for disabled should be `!activeStep && !skippedIndex && (completed + 1 < index)`